src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/inventory/Item.kt

```diff
@@ -127,7 +131,15 @@ class Item {
      *
      * @return the item's stack display name
      */
-    fun getName(): String = if (getID() == 0) "air" else itemStack.displayName
+    fun getName(): String = itemStack.displayName
```

Comment:
Is this breaking as well? What is air's `displayName`?

Reply:
Air is technically null, so an Item can't be made out of air since Item not take null Item stacks.  The check was useless when this was changed.  I added the checks to the constructors since if you tried to do new Item(0) it would cause an internal error, since that would make the Item stack null.